controllers/operatorpipeline_controller.go

```diff
@@ -63,12 +64,29 @@ func (r *OperatorPipelineReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return ctrl.Result{}, err
 	}
 
-	err = r.reconcileResources(ctx, pipeline)
+	err = r.reconcileResources(ctx, currentPipeline)
 	if err != nil {
 		return ctrl.Result{}, err
 	}
 
-	return ctrl.Result{}, nil
+	reconcilers := []reconcilers.Reconciler{
+		reconcilers.NewCertifiedImageStreamReconciler(r.Client, reqLogger, r.Scheme),
+		reconcilers.NewMarketplaceImageStreamReconciler(r.Client, reqLogger, r.Scheme),
+		reconcilers.NewStatusReconciler(r.Client, reqLogger, r.Scheme),
+	}
+
+	requeueResult := false
+	pipeline := currentPipeline.DeepCopy()
+	for _, r := range reconcilers {
+		requeue, err := r.Reconcile(ctx, pipeline)
```

Comment:
if/when the reconcilers list gets longer in the future, we could use Goroutines here to launch them.